Udn

HadleyKing opened the pull request.

Commit:
Initial udn server path dup

Commit:
Patched data converter to use create methods instead of create_or_update

Commit:
UDN import works as expected now. Typo in genetic findings model for public_database_other did not allow for arrays

Commit:
Added selector options for multi-value fields partial_contribution_explained and additional_family_members_with_variant

Commit:
Added phenotype multi-value field parsing

Commit:
Revert error handling after importing HPO additional terms

HadleyKing committed:
Fixes for Biobank model

Changes to be committed:
	modified:   server/metadata/apis.py
	modified:   server/metadata/models.py
	modified:   server/metadata/services.py
	modified:   server/tests/fixtures/test_fixture.json
	modified:   server/tests/test_apis/test_biobank_apis.py

HadleyKing committed:
Model update for biobank

Changes to be committed:
	modified:   server/metadata/models.py
	modified:   server/metadata/selectors.py
	modified:   server/metadata/services.py
	modified:   server/tests/fixtures/test_fixture.json

Commit:
Added completed field to biobank json schema

Commit:
Re-versioned pip requirements

IvantheDugtrio reviewed (dismissed):
Will check if pip dependency upgrades break anything outside of testing. This will help patch Django vulnerabilities noted on the dev branch.

HadleyKing committed:
Fixes for BioBank import

Changes to be committed:
	new file:   server/experiments/migrations/0001_initial.py
	new file:   server/experiments/migrations/0002_initial.py
	new file:   server/experiments/migrations/__init__.py
	new file:   server/metadata/migrations/0001_initial.py
	new file:   server/metadata/migrations/__init__.py
	modified:   server/submodels/migrations/0001_initial.py
	deleted:    server/submodels/migrations/0002_alignedid_experimentid.py
	new file:   server/utilities/safe_loader.py

HadleyKing committed:
Roll back `requirements.txt`

Changes to be committed:
	modified:   server/requirements.txt

HadleyKing committed:
Update migrations and fix `requirements.txt`

Changes to be committed:
	new file:   server/experiments/migrations/0001_initial.py
	new file:   server/experiments/migrations/0002_initial.py
	new file:   server/experiments/migrations/__init__.py
	new file:   server/metadata/migrations/0001_initial.py
	new file:   server/metadata/migrations/__init__.py
	modified:   server/requirements.txt
	new file:   server/utilities/safe_loader.py
